composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
Please fix the build when you have some time, that way we will see if it really is that much longer. I think we now have cache we did not have before.